src/adera/Machines/Rocket.cpp

```diff
@@ -60,6 +60,15 @@ WireOutput* MachineRocket::request_output(WireOutPort port)
 
 std::vector<WireInput*> MachineRocket::existing_inputs()
 {
+    /*std::vector<WireInput*> inputs;
+    inputs.reserve(3 + m_resourceLines.size());
+
+    inputs.insert(inputs.begin(), {&m_wiGimbal, &m_wiIgnition, &m_wiThrottle});
+    for (auto& resource : m_resourceLines)
+    {
+        inputs.push_back(&resource.m_lineIn);
+    }
+    return inputs;*/
```

Comment:
prefer
```
#if 0
#endif 
```
for commenting out code.

Reply:
or #if false

Reply:
Really? Why?

Reply:
Much easier for a lot of code editors to handle gracefully, in addition to not having weird behavior when it comes to nested comments.